templates/ksql/filtered/code/.gitignore

```diff
@@ -0,0 +1,4 @@
+extensions/
+tutorial-steps/dev/outputs/
+tutorial-steps/test/outputs/
+tutorial-steps/prod/outputs/
```

Comment:
I think this file is extraneous. We can remove it, right?

`templates/ksql/filtering/code/.gitignore` was also created.

Reply:
I was under the assumption that this 'filtered' directory is the template used to create new tutorials, while `_includes/tutorials/filtering`, (which is what I assume you're referring to, though not the different root), is a tutorial instance.

I'm going to merge this PR - if I'm wrong and this file is superfluous, then please feel free to remove it!